sonar-php-plugin/src/test/java/org/sonar/plugins/php/PHPSensorTest.java

```diff
@@ -179,14 +177,44 @@ private void checkNoSonar(String componentKey, int line, boolean expected) throw
   }
 
   @Test
-  public void parse_error() throws Exception {
-    SensorContext context = mock(SensorContext.class);
-    when(context.settings()).thenReturn(new Settings());
+  public void empty_file_should_raise_no_issue() throws Exception {
```

Comment:
@vilchik-elena You're right, thanks. I will make sure that the rule is activated.